### ISSUE TITLE
Meteor support or tutorial?

### ISSUE THREAD

**Issue by insaneinc01** [reapp/reapp#98]
Reapp looks like the only full-fledged react based mobile UI kit out there. And its looks pretty amazing!

Is there a way to use reapp-ui with Meteor? I know https://github.com/reapp/reapp-ui is the way to go, but i'm not sure if the official meteor react support would play nice with adding reapp-ui into the project.
Thanks.

**Comment by mbrookes** [comment]
Meteor's React support just enables the use of React in Meteor, so in principal, with a little bit of work, this should drop right in. Where things might get tricky is around routing and transition animations.

**Comment by insaneinc01** [comment]
Thanks for your quick reply! 
In fact I'm trying to make Meteor + React + Reapp work and am stuck exactly there: **routing** and **screen transition animations**. :|

I was wondering if anyone else got it working smoothly and looking for some steps. 
Also, I have come across this on NPM - https://www.npmjs.com/package/reapp-ui-meteorcompat but not sure how it would help.

**Comment by mbrookes** [comment]
I don't think that package does anything other than make reapp-ui available via npm, which might make install marginally easier, but not much else! 

Sorry I can't help more on your issue, but if you can be more specific about what what you've done, and where you're stuck (and maybe a github repo to look at), someone might chip in. there's also a (relatively quiet) channel in gitter.im, but a few knowledgable folks there.

**Comment by hellogerard** [comment]
I have been looking at this as well, without success :-(

There are 2 approaches to integrating Reapp with Meteor.

1. Create a wrapper Meteor package depending on the `reapp-ui` npm module. The [`izzilab:material-ui`](https://github.com/mrphu3074/react-material-ui)Meteor package does this for Material UI.  I attempted this and the 1st issue I ran into was Meteor complaining about an icon file with a `:` (colon) in the filename. That npm module mentioned above (`reapp-ui-meteorcompat`) fixes this issue. After using that, I can't get the browserify step to see Reapp at all. That's where I'm stuck. I'm afraid I do not know much about building JS things. I am trying to emulate that Material UI Meteor package but no luck.

2. Use webpack via [`webpack:webpack`](https://github.com/thereactivestack/meteor-webpack) which integrates Webpack with Meteor. This appears to be working, but Reapp themes are not working at all, so any component I attempt to render ends up as plain DIVs.

I've got experimental code spread all over, but I can pull it into a github repo if there is interest.

**Comment by hellogerard** [comment]
Update: Focusing on #2, I put `reapp-ui` into this example project (https://github.com/gaearon/react-hot-boilerplate) and I am getting the same issue of no themes. So it may not be a Meteor issue. Since this has turned into a `reapp-ui` specific effort, I am moving over there.

**Comment by hellogerard** [comment]
_I had a couple posts on here that I just deleted because they were rambling and probably only confused the issue even more._

I have `reapp-ui` (minimally) loaded into Meteor using the `webpack:webpack` Meteor package. I was stumped initially by, I believe, an unrelated theme bug that [appears to be fixed now](https://github.com/reapp/reapp-ui/issues/83).

So I now seeing Reapp components and themes. I am trying to get the kitchen sink into Meteor, but I am getting a blank screen at the moment. It's probably an issue with the router, so I think next step is to discard the `reapp-routes` package and go to `react-router` directly.

**Comment by raarts** [comment]
@hellogerard : I am currently in the same position. Did you make any progress?

**Comment by hellogerard** [comment]
@raarts Not really. But the conversation proceeded just a little bit further on the Meteor forums: https://forums.meteor.com/t/meteor-react-reapp/12003/10?u=hellogerard
